unskript-ctl/unskript-client.py

```diff
@@ -912,7 +1005,29 @@ def display_failed_checks(args):
     print("")
 
 
-def display_failed_logs(exec_id: str = None):
+def display_failed_logs(args):
+
+    parser = ArgumentParser(description='-dl | --display-failed-logs')
+
+    parser.add_argument('-dl', 
+                        '--display-failed-logs',
+                        help=SUPPRESS,
+                        required=True,
+                        default=True,
+                        action="store_true")
+    parser.add_argument('--execution_id',
+                        help="Execution ID for which the Logs should be fetched",
+                        type=str)
+    
+    args = parser.parse_args()
+    
+    if args.execution_id not in ('', None):
```

Comment:
need num args check as well, otherwise -dl wont show the help string

Reply:
Done